README.md

```diff
@@ -1 +1,59 @@
-# todo_app
\ No newline at end of file
+# TodoApp
+
+Todo application created in React, ExpressJS and mongoDB with CRUD(create, read, update, delete) functionality.
+
+## Installation
+
+Clone the repository:
+
+```
+https://github.com/vladimirmomiroski/todo_app.git
+```
+
+Install the dependencies: 
+
+```
+npm install --prefix ./backend
+```
+```
+npm install --prefix ./frontend
+```
+
+## Setup
+
+Create a .env.local file inside frontend directory:
```

Comment:
Provide example of database link without credentials if you are using mongo example of the link 
DB_URL = mongo<user><password>